packages/shared/src/hooks/useContextMenu.ts

```diff
@@ -1,15 +1,18 @@
+import { useState } from 'react';
 import { useContextMenu as useContexifyContextMenu } from '@dailydotdev/react-contexify';
 
 export default function useContextMenu({ id }: { id: string }): {
   onMenuClick: (e: React.MouseEvent) => void;
 } {
+  const [menu, setMenu] = useState(true);
```

Comment:
The default value should be false.
```suggestion
  const [menu, setMenu] = useState(false);
```